CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - `wfx`: implemented sort functionality for `/workflows` endpoint
+- `wfx`: fixed an issue where job event connections were prematurely closed due to inactivity
```

Comment:
Please also state the impact: The job execution is disturbed / broken.

Reply:
Hm, I didn't observe that behavior. `wfxctl` exited due to the connection being closed, but `wfx` didn't care: I was still able to update jobs using `wfxctl`, also `CTRL+c` resulted in a clean shutdown of wfx.